src/services/AuthService.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+import { url } from '../components/constants';
+// import url from "../http";
+
+axios.defaults.withCredentials = true;
+
+export const registration = async (login, password) => await axios.post(`${url}/registration`, { login: login, password: password });
```

Comment:
firstly, why is async / await here when the functions are already asynchronous? Second, use the abbreviation
{ login, password }